### DISCUSSION TITLE
Não consigo fazer a execução do debug - Curso Fundamento de C#, cap .NEt, aula debug

### DISCUSSION THREAD

**Discussion by marley2santos** [balta-io/forum#353] [Tira Dúvidas] [answered]
![image](https://github.com/balta-io/forum/assets/72535842/96896e6e-4787-468c-9c7b-bfba3bfd9dc6)

basicamente quando clico na parte da depuração, não há opção para rodar. 

![image](https://github.com/balta-io/forum/assets/72535842/57fdd80e-6169-4690-b1f2-5e32ddcb33ac)

Eu clico no Run and Debug e abre a caixa de pesquisa. Sendo que já está instalado a extensão pra C#. Daí até aparece la embaixo um ícone de play mas ao clicar aparece isso ai:

![image](https://github.com/balta-io/forum/assets/72535842/ea9d00b8-7a5e-4e9b-83d7-5293ccfb7b36)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @marley2santos, como vai?

Você está usando qual versão do .NET? A 5 igual do curso?
Instalou as extensões necessárias no Visual Studio Code?

Em adicional, após clicar em Debug, se você selecionar a opção ".NET 5+ and .NET Core" que aparece na sua segunda imagem, ele não inicia o Debug?
